entry_types/scrolled/package/src/contentElements/videoEmbed/VideoEmbed.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactPlayer from 'react-player';
+import styles from './VideoEmbed.module.css';
+import classNames from 'classnames';
+import {InlineCaption} from "../inlineImage/InlineCaption";
```

Comment:
This is fine for now, but generally I'd be careful to introduce too many inter-dependencies between content elements. Maybe a variation of the rule of three: Copy to new content element on first reuse. Extract to `pageflow-scrolled/frontend` (and test and document) on second reuse.